source/_integrations/matter.markdown

```diff
@@ -290,7 +290,7 @@ Also see this [extended troubleshooting guide](https://developers.home.google.co
 
 ### Unable to commission devices, it keeps giving errors or stops working randomly
 
-The Matter protocol relies on (local) IPv6 and mDNS (multicast traffic) which should be able to travel freely in your network. Matter devices that use Wi-Fi (including Thread Border routers) must be on the same LAN/VLAN as Home Assistant. Matter devices that only use {% term Thread %} must be joined to {% term Thread %} networks for which there is at least one border router connected to the Home Assistant LAN.
+The Matter protocol relies on (local) IPv6 and mDNS (multicast traffic) which should be able to travel freely in your network. Matter devices that use Wi-Fi (including {% term Thread border routers %}) must be on the same LAN/VLAN as Home Assistant. Matter devices that only use {% term Thread %} must be joined to {% term Thread %} networks for which there is at least one border router connected to the Home Assistant LAN.
```

Comment:
Does the term take plural (routers) too?

Reply:
right. that is the problem. Thank you!

Reply:
Why is that PR needed?

Reply:
Haha, :see_no_evil: it doesn't. I thought I had mixed in in there. But I hadn't saved the file. So, the alias got its proper PR now. #31054. Thank you for bearing with me.